packages/compiler/src/core/parser.ts

```diff
@@ -2513,13 +2539,10 @@ function createParser(code: string | SourceFile, options: ParseOptions = {}): Pa
     const items: T[] = [];
     while (true) {
       let pos = tokenPos();
-      let docs: DocNode[] | undefined;
 
-      if (!kind.invalidAnnotationTarget) {
-        [pos, docs] = parseDocList();
-      }
-      const directives = parseDirectiveList();
-      const decorators = parseDecoratorList();
+      // todo need to disable doc parsing? if kind.invalidAnnotationTarget
```

Comment:
@nguerrera what was the reason for not parsing the doc list here? should I add a new parameter to parseAnnotation to disable it?

Tests seems to be passing without.

Reply:
ok added option